instaclustr/structs.go

```diff
@@ -95,15 +95,15 @@ type Cluster struct {
 	ClusterStatus              string        	`json:"clusterStatus"`
 	BundleType                 string        	`json:"bundleType"`
 	BundleVersion              string        	`json:"bundleVersion"`
-	AddonBundles			   map[string]interface{}	 `json:"addonBundles"`
+	AddonBundles			   []map[string]interface{}	 `json:"addonBundles"`
 	Username                   string        	`json:"username"`
 	InstaclustrUserPassword    string        	`json:"instaclustrUserPassword"`
 	SlaTier                    string        	`json:"slaTier"`
 	ClusterCertificateDownload string        	`json:"clusterCertificateDownload"`
 	PciCompliance              string        	`json:"pciCompliance"`
 	BundleOption               *BundleOptions 	`json:"bundleOptions"`
 	DataCentres                []DataCentre  	`json:"dataCentres"`
-	Provider				   ClusterProvider	`json:"clusterProvider"`
+	Provider				   []ClusterProvider	`json:"clusterProvider"`
```

Comment:
**[Question]** I don't understand why there would be more than one `ClusterProvider`. Could you clarify this for me please?

Reply:
Talked to Micheal and we do support multiple providers for a single cluster. Though will need to change the state updater in read to handle this instead of just the first element.